spec/requests/v0/caregivers_assistance_claims_spec.rb

```diff
@@ -189,75 +189,158 @@
       FileUtils.rm_f(response_pdf)
     end
 
-    it 'returns a completed PDF', run_at: '2017-07-25 00:00:00 -0400' do
-      expect(SavedClaim::CaregiversAssistanceClaim).to receive(:new).with(
-        form: form_data
-      ).and_return(
-        claim
-      )
+    context 'caregiver1010 toggle off' do
+      before do
+        allow(Flipper).to receive(:enabled?).with(:caregiver1010).and_return(false)
+      end
```

Comment:
Consider this approach in the future, it's just a little easier to read:

```ruby
before do
  Flipper.enable(:caregiver1010)
end
```

Reply:
🤔 Doesn't this leak into other tests run in the same batch?

Reply:
Yeah, unless you add 
```
after do
  Flipper.disable(:caregiver1010)
end
```
which is why I tend to use the allow instead of directly enabling/disabling. I agree it's more verbose so not as obvious, though.